src/SourceBuild/patches/emsdk/0001-Move-NoTargetFrameworkFiltering-property.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Mon, 24 Apr 2023 14:00:42 +0000
+Subject: [PATCH] Move NoTargetFrameworkFiltering property
+
+Backport: https://github.com/dotnet/source-build/issues/3417
+---
+ Directory.Build.props | 5 +++++
+ eng/SourceBuild.props | 1 -
+ 2 files changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/Directory.Build.props b/Directory.Build.props
+index 69862d8..defa3f0 100644
+--- a/Directory.Build.props
++++ b/Directory.Build.props
+@@ -2,6 +2,11 @@
+ <Project>
+   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+ 
++  <PropertyGroup>
++    <!-- Emsdk doesn't support Arcade-driven target framework filtering. -->
++    <NoTargetFrameworkFiltering>true</NoTargetFrameworkFiltering>
```

Comment:
I am curious why this needs to be set in the repo's root Directory.Build.props file.  It seems preferrable in the SourceBuild.props.